### PR TITLE
Add v0.5.0 of mattermost-plugin-apps to the Marketplace

### PR DESCRIPTION
**Pull request by mickmister**

### CODE REVIEW

File: plugins.json [hunk 1]
```diff
@@ -2155,6 +2155,56 @@
     },
     "updated_at": "2020-11-20T18:12:44.537612606Z"
   },
+  {
+    "homepage_url": "https://github.com/mattermost/mattermost-plugin-apps",
```
Comment (catalintomai) [inline plugins.json:2159]: @mickmister , do we need an update to the statik.go file?

**Comment by DHaussermann** [comment]
This changed passed functional testing. 
- Upgrade path works - tested manually(not with split io) 
- Install path works
- Pluign correctly hidden for non-Cloud license

LGTM!

Will retest if needed.

**Review by DHaussermann — APPROVED** [review]
Approved. 

Please re-request if there are additional code changes

File: plugins.json (hunk 1 above)
Comment (mickmister) [reply plugins.json:2159]: @catalintomai the file is no longer part of the repo, neither is the `make generate` command. I'm thinking CI does the replacement on deployment of the marketplace now

Comment (mickmister) [reply plugins.json:2159]: I believe it's meant to facilitate a faster parsing of the plugins data on a cold start of the marketplace lambda function

**Review by catalintomai — APPROVED** [review]

**Comment by aaronrothschild** [comment]
@hanzei @DHaussermann Shold the Min server version be set to 5.34 (or something cloud release) so when it is on the marketplace, it isn't showing up for old servers without the webapp code in it?

**Comment by hanzei** [comment]
@aaronrothschild This release is only available for cloud so no need to care about old servers. For next on-prem release (0.6.0) `v5.35.0` will be required. See https://github.com/mattermost/mattermost-plugin-apps/pull/143

File: plugins.json (hunk 1 above)
Comment (hanzei) [reply plugins.json:2159]: `go embed` is used to include the plugin list: https://github.com/mattermost/mattermost-marketplace/pull/170

**Comment by mickmister** [comment]
@hanzei The README of this repo says

> When adding or updating the plugins database (or corresponding tooling), submit the changes directly to `production`, and then merge `production` immediately back to `master` to reduce unnecessary merge conflicts.

Should this be done through a follow-up PR every time we add a new plugin to the marketplace?

**Comment by hanzei** [comment]
Hu, I didn't know that part exists. I'm inclined to keep the process as it is. What do you think?

**Comment by mickmister** [comment]
@hanzei If it ain't broke don't fix it :wink:  I'm not sure what the tradeoffs are. The merge we recently did was pretty unreadable, but idk if this would solve that

**Pull request merged**